### PR TITLE
Add German CONTRIBUTING

### PR DESCRIPTION
**Pull request by Jean-Sim**

### CODE REVIEW

**Comment by Jean-Sim** [comment]
ok the language lists should be in alphabetical order now.

File: CONTRIBUTING.de.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+[Kate]:https://github.com/virtualagc/virtualagc/tree/master/Contributed/SyntaxHighlight/Kate
+[ProgrammersNotepad]:https://github.com/virtualagc/virtualagc/tree/master/Contributed/SyntaxHighlight/ProgrammersNotepad
+[Sublime Text]:https://github.com/jimlawton/AGC-Assembly
+[TextPad]:https://github.com/virtualagc/virtualagc/tree/master/Contributed/SyntaxHighlight/TextPad
+[Vim]:https://github.com/wsdjeg/vim-assembly
+[VisualStudioCode]:https://github.com/wopian/agc-assembly
+[jEdit]:https://github.com/virtualagc/virtualagc/tree/master/Contributed/SyntaxHighlight/jEdit
+
+## Formatierung  
+**Anmerkung:** Die oben erwähnten GitHub Erweiterungen werden automatisch sicherstellen dass sie die Korrekte Formatierung einhalten.
```
Comment (wopian) [inline CONTRIBUTING.de.md:61]: Is this "GitHub extensions mentioned above"? If it is, it should be "GitHub **and** [the] extensions mentioned above.

Just basing this of Google Translate as I don't know German.

**Comment by Jean-Sim** [comment]
I checked the sentence and you are correct. What I forgot was the 'and' between 'GitHub' and 'extensions'. It Should be translated correct now.

**Comment by chrislgarry** [comment]
Thanks!

**Pull request merged**